packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/java/sdk.md

```diff
@@ -104,8 +104,8 @@ public ModelAndView handleKnownTransitions(
 
 Display the password reset page and continue the password recovery flow described in the [User password recovery summary of steps](/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/java/main/#summary-of-steps).
 
-<div class="common-image-format">
```

Comment:
Image has got some drop-shadows going on in it. Need to remove those or wait for wireframe?

Reply:
drop shadow removed and CSS border added. Looks pretty decent now.

Reply:
very nice